experiments/dataSources/worldBank/readme.md

```diff
@@ -0,0 +1,102 @@
+# Analyse World Bank statictical data in Skuid
+The World Bank provides free and open access to global development data.  Information like GDP, Population, Economic growth and Development measures are available for your Skuid pages. This is one of the easiest API to interact with. 
+
+Some introductory links with information about this API from the World Bank Website: 
+-  [Overview of the data available through this API](https://datahelpdesk.worldbank.org/knowledgebase/articles/889386-developer-information-overview) 
+
+- [About the API:](https://datahelpdesk.worldbank.org/knowledgebase/articles/889392)
+
+- [The Basic Call Structure:](https://datahelpdesk.worldbank.org/knowledgebase/articles/898581)
+
+
+This API is exposed without any authentication - which makes it super easy to use in Skuid. It is a great way to explore the following concepts in Skuid:
+- Searching / filtering for a particular record.
+- Using stastical data in dashboards. 
+
+## How to use this data source
+
+### Create the data source
+
+First you'll need to create a new REST data source. 
+* Type: REST
+* Name: WorldBank
+* URL / Endpoint: ``http://api.worldbank.org/v2/``
+* Use Proxy: False
+* Authentication Method: No Authentication
+
+
+#### Double check the remote site setting
+
+* If you are using Skuid on Salesforce, you will be prompted to appove the creation of a remote site setting for this data source. If your pages do not work, you might need to edit it.  Check in Salesforce Setup at **Security > Remote site settings**.   
+
+#### Create a CORS entry
+
+* Allowing the browser to retrieve data directly is the fastest method for data sources (as opposed to using the Server Proxy), but if you are using Skuid on Salesforce, you will be required that you whitelist the endpoint so you don't have a CORS violation.
+* Within Salesforce Setup, go to **Security > CORS**.
+* Create a new entry - generally it will be the same domain where you ultimately intend to host the page. ``https://<<mydomain>>.visual.force.com``
+
+### Create a page with this REST data source to review World Bank statistics
+
```

Comment:
Man... This feels like a lot to do just to get a working model, especially when we already have nice sample pages. Do you think it'd be better to put the Sample Pages section *above* this, and then say "if you want a technical explanation of how we set up these models, read below"?

Reply:
I'm going to leave this order, but add a parenthetical note below the "create a new page" line.